### PR TITLE
Stringtables

### PR DESCRIPTION
**Pull request by jokoho48**
### When merged this pull request will:

1. *Do Stringtable*

### CODE REVIEW

**Comment by jokoho48** [comment]
Total number of keys: 102

| Language | Missing Entries | Relevant Modules | % done |
|----------|----------------:|------------------|--------|
| English | 0 | - | 100% |
| German | 101 | danger, eventhandlers, main, wp | 1% |
| Polish | 101 | danger, eventhandlers, main, wp | 1% |
| Portuguese | 101 | danger, eventhandlers, main, wp | 1% |
| Russian | 101 | danger, eventhandlers, main, wp | 1% |
| Czech | 101 | danger, eventhandlers, main, wp | 1% |
| Spanish | 101 | danger, eventhandlers, main, wp | 1% |
| Italian | 101 | danger, eventhandlers, main, wp | 1% |
| French | 101 | danger, eventhandlers, main, wp | 1% |
| Korean | 101 | danger, eventhandlers, main, wp | 1% |
| Japanese | 101 | danger, eventhandlers, main, wp | 1% |
| Chinese | 101 | danger, eventhandlers, main, wp | 1% |
| Chinesesimp | 101 | danger, eventhandlers, main, wp | 1% |

**Comment by diwako** [comment]
Found some forgotten strings:

https://github.com/nk3nny/LambsDanger/blob/e4092945fb3862c15da62c2856ad1d41611c36d9/addons/main/functions/fnc_showDialog.sqf#L214


https://github.com/nk3nny/LambsDanger/blob/e4092945fb3862c15da62c2856ad1d41611c36d9/addons/main/functions/fnc_showDialog.sqf#L227

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            <English>Script interval</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskAssault_CycleTime_Tooltip">
+            <English>The cycle time of the script</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCamp_DisplayName">
+            <English>Task Camp</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCamp_ModuleDescription">
+            <English>Task Camp makes the unit appear to gather around a single point, typically a campsite. Some units will peel off to man static weapons, garrison buildings and patrol the area.<br/>Set area in 3DEN.</English>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: <br/> needs to be escaped

File: addons/wp/stringtable.xml [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+            <English>With the dynamic center present, it is possible to move the center point of the building search pattern</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCQB_Radius_DisplayName">
+            <English>Radius</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCQB_Radius_ToolTip">
+            <English>Max distance houses will be searched</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCQB_ModuleDescription">
+            <English>Task CQB makes the unit search buildings with the area. Set the module size in 3DEN to configure the search area. The module can be moved to dynamically shift the searched area.<br/><br/>In general the module works best with a smaller AOE. Better to string multiple modules or waypoints together than have one massive one. Groups of the same side will share which buildings have been cleared and which still need searching.</English>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: <br/> needs to be escaped

File: addons/wp/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+            <English>The cycle time for the script in seconds. Higher numbers can be used to make the creeping group less accurate.\nDefault 15 seconds</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_Radius_DisplayName">
+            <English>Radius</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_Radius_ToolTip">
+            <English>Distance creeping group will search for enemies.\nThis module only targets enemy players</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_ModuleDescription">
+            <English>Task Creep makes the unit attempt to ambush enemies within the area set by 3DEN attributes. The group will use available cover and keep a low stance to remain undiscovered.<br/><br/>Search modules only target hostile players</English>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: <br/> needs to be escaped

File: addons/wp/stringtable.xml [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_DisplayName">
+            <English>Task Hunt</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_ModuleDescription">
+            <English>Task Hunt makes the unit patrol the area within the area set by 3DEN attributes. The group will move casually in a search pattern gradually spiraling closer to the target. The unit will enable flashlights if available.<br/><br/>Search modules only target hostile players.</English>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: <br/> needs to be escaped

File: addons/wp/stringtable.xml [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+            <English>Radius</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_Radius_ToolTip">
+            <English>Distance hunting group will search for enemies.\nThis module only targets enemy players</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskRush_DisplayName">
+            <English>Task Rush</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskRush_ModuleDescription">
+            <English>Task Rush makes the unit attempt to run aggressively towards enemies within the area set by 3DEN attributes. The unit will move extremely aggressively, often ignoring cover and suppression.<br/><br/>Search modules only target hostile players. Disable unit FSM for a truly terrifying enemy.</English>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: <br/> needs to be escaped

**Pull request merged**